app/models/service_template_provision_task.rb

```diff
@@ -169,6 +169,14 @@ def mark_pending_items_as_finished
     end
   end
 
+  def before_ae_starts(_options)
+    reload
+    unless state.to_s.downcase == "active"
```

Comment:
After some discussion we feel it is appropriate to always set the state to `active` here.